schema.json

```
@@ -1,329 +1,423 @@
-             }
-         },
-         {
-             "name":"type_depense",
-             "description":"Liste des types de dépenses. Deux options : \"Fonctionnement\", \"Investissement\". S'il y a plusieurs types de dépenses, mettre une virgule sans espace entre les différents types de dépense.",
-             "example": "Fonctionnement,Investissement",
-             "type":"string",
-             "constraints":{
-                "required":false,
-                "pattern": "(?:(?:^|,)(Fonctionnement|Investissement))+$"
-             }
-         },
-         {
-             "name":"zone_geographique",
-             "description":"Zone géographique couverte par l'aide.",
-             "example": "Tarn",
-             "type":"string",
-             "constraints":{
-                "required":true
-             }
-         },
-         {
-             "name":"url_descriptif",
-             "description":"Lien vers le descriptif complet.",
-             "example": "https://aides-territoires.beta.gouv.fr/aides/7ee8-creer-un-agregateur-territorial-a-la-rochelle",
-             "type":"string",
-             "format":"uri"
-         },
-         {
-             "name":"url_demarche",
-             "description":"Lien vers la démarche en ligne.",
-             "example": "https://aides-territoires.beta.gouv.fr/aides/7ee8-creer-un-agregateur-territorial-a-la-rochelle",
-             "type":"string",
-             "format":"uri"
-         },
-         {
-             "name":"contact",
-             "description":"Contact(s) pour candidater.",
-             "example": "Ecrire un email à aides@programme.gouv.fr",
-             "type":"string",
-             "constraints":{
-                "required":true
-             }
-         },
-         {
-             "name":"date_mise_a_jour",
-             "description":"Date de mise à jour du jeu de données.",
-             "example": "2022-03-03",
-             "type":"date",
-             "format":"%Y-%m-%d"
-         }
-    ]
- }
+  "$schema": "https://specs.frictionlessdata.io/schemas/table-schema.json",
+  "name": "dispositif-aide",
+  "title": "Dispositifs d'aides",
+  "description": "Spécification du fichier d'échange relatif aux dispositifs d'aides.",
+  "keywords": [
+    "dispositifs",
+    "aide"
+  ],
+  "countryCode": "FR",
+  "homepage": "https://github.com/etalab/schema-dispositif-aide",
+  "path": "https://github.com/etalab/schema-dispositif-aide/raw/v0.1.0/schema.json",
+  "licenses": [
+    {
+      "title": "Etalab Licence Ouverte 2.0",
+      "name": "etalab-2.0",
+      "path": "https://spdx.org/licenses/etalab-2.0.html"
+    }
+  ],
+  "resources": [
+    {
+      "title": "Fichier valide (CSV)",
+      "name": "exemple-valide-csv",
+      "path": "https://github.com/etalab/schema-dispositif-aide/raw/v0.1.0/exemple-valide.csv"
+    }
+  ],
+  "sources": [],
+  "created": "2022-03-14T00:00:00Z",
+  "lastModified": "2025-05-28T00:00:00Z",
```

On a l'habitude de renseigner simplement la date
```suggestion
  "created": "2022-03-14",
  "lastModified": "2025-05-28",
```